spec/lib/administrate/search_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "spec_helper"
+require "support/constant_helpers"
+require "administrate/fields/string"
+require "administrate/fields/email"
+require "administrate/fields/number"
+require "administrate/search"
+
+class MockDashboard
+  ATTRIBUTE_TYPES = {
+    name: Administrate::Field::String,
+    email: Administrate::Field::Email,
+    phone: Administrate::Field::Number,
+  }
+end
+
+describe Administrate::Search do
+  describe "#run" do
+    let(:resolver) do
```

Comment:
[thoughtbot's guides][guides] recommend avoiding `let` - it can be easy to lose track of where variables are being defined in specs that have a lot of `let` definitions, because it removes variables from the specs in which they're used.

[Here's an example of an alternative][avoid let]. How would you feel about moving the variable definition into the individual specs?

[guides]: https://github.com/thoughtbot/guides
[avoid let]: https://github.com/thoughtbot/guides/blob/master/style/testing/avoid_let_spec.rb